### ISSUE TITLE
Error when querying an empty table with unique constraints

### ISSUE THREAD

**Issue by happy-river** [wotbrew/relic#59]
Including the following expression in my ClojureScript project:

```
(-> {}
    (rel/mat [[:from :Thing] [:unique :a :b]])
    (rel/transact [:insert :Thing {:a 1 :b 2}])
    (rel/transact [:delete :Thing [= :a 1] [= :b 2]])
    (rel/q [[:from :Thing] [:where [= :a 1] [= :b 2]]]))
```

causes this error in the browser console:

```
Uncaught TypeError: Cannot read properties of null (reading 'cljs$core$IFn$_invoke$arity$1')
    at eval (dataflow.cljc:1368:61)
    at Function.G__123169__2 [as cljs$core$IFn$_invoke$arity$2] (core.cljs:4757:24)
    at eval (core.cljs:10843:15)
    at eval (core.cljs:5686:36)
    at eval (core.cljs:5686:35)
    at Object.eval [as cljs$core$IReduce$_reduce$arity$3] (core.cljs:5690:24)
    at Function.eval [as cljs$core$IFn$_invoke$arity$3] (core.cljs:2570:17)
    at Function.G__124171__2 [as cljs$core$IFn$_invoke$arity$2] (core.cljs:10858:11)
    at Function.G__123169__2 [as cljs$core$IFn$_invoke$arity$2] (core.cljs:4757:13)
at eval (core.cljs:4200:31)
```

**Comment by wotbrew** [comment]
Thanks for the report @happy-river, I believe this is fixed in `master` but will test the repro and work on a patch release.

**Comment by wotbrew** [comment]
Fixed in 0.1.7
